src/tools/haxelib/Main.hx

```diff
@@ -977,11 +981,13 @@ class Main {
 				} catch( e : String ) {
 					throw "Error accessing Haxelib repository: $e";
 				}
-				return Path.addTrailingSlash( rep );
+				return [Path.addTrailingSlash( rep )];
 			} else
 				throw "This is the first time you are runing haxelib. Please run `haxelib setup` first";
 		}
 		rep = rep.trim();
```

Comment:
yes but don't the string still needs trimming first before splitting ? What if the value is ```"       /haxelib/path:/other/haxelib/path      "``` ?